app/components/Settings/UserConnections/UserConnections.react.js

```diff
@@ -57,9 +66,22 @@ export default class UserConnections extends Component {
 	render() {
 
 		const {connectionObject, updateConnection} = this.props;
-		const inputs = CONNECTION_CONFIG[connectionObject.dialect]
+        const sampleCredentialsStyle = {
+            lineHeight: 1,
+            textAlign: 'left',
+            width: '100%',
+            maxWidth: 200,
+            display: 'inline-block'
+        };
+
+        if (!this.state.showSampleCredentials) {
+            sampleCredentialsStyle['display'] = 'none';
+        }
```

Comment:
it looks like these lines need to be indented. On github, it looks like they are in-line with `render`